swan-lake/integration-tools/consolidate-packages-tool.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: ballerina-consolidate-packages-support-left-nav-pages-swanlake
+title: Consolidate-packages tool
+description: The consolidate-packages tool provides CLI commands and a build tool to consolidate services into a single executable.
+keywords: ballerina, programming language, consolidate packages, consolidate-packages, monolith
+permalink: /learn/consolidate-packages-tool/
+active: consolidate-packages-tool
+intro: This tool generates code required to group multiple services into one deployable artifact supporting monolith-style deployments.
+---
+
+Ballerina inherently supports microservices-style deployments, which are well-suited for microservice orchestration platforms like Kubernetes.
+However, many enterprise users deploy on VMs or Docker, where managing each service as a separate process increases complexity and resource overhead. 
+This tool provides a solution tailored to such scenarios by enabling consolidated deployments.
+
+## Usage
+
+### Create or update a consolidator package
+
+Create a new package and add the `consolidate-packages` tool entry as shown in the below example in the `Ballerina.toml` file with the services required to consolidate. Services can be added or removed as needed by updating the values provided for the `options.services` array. The tool will be automatically pulled during the package build.
+
+```toml
+[package]
+org = "myorg"
+name = "myapp"
+version = "0.1.0"
+
+[[tool.consolidate-packages]]
+id = "consolidateSvc"
+options.services = ["myorg/svc1", "myorg/svc2"]
+```
+
+#### Using the CLI tool
+Alternatively, the `consolidate-packages` CLI tool can be installed to create the package and add or remove services. This
+is typically useful in CI/CD pipelines.
```

Comment:
Was this line break added on purpose?

Reply:
No, fixed in [551f659](https://github.com/ballerina-platform/ballerina-dev-website/pull/9738/commits/551f6591d061d334b77b363a8b9fb70c3ec1adfe)